Resources/config/exception_listener.xml

```diff
@@ -5,13 +5,22 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="fos_rest.exception_listener" class="FOS\RestBundle\EventListener\ExceptionListener">
-            <tag name="kernel.event_subscriber" />
+        <service id="security.exception_listener" class="FOS\RestBundle\EventListener\ExceptionListener">
```

Comment:
OK, well I'm a little stuck now how to move forwards with this one. I'll await feedback because I'm not 100% sure if this is the best way or if there should just be the option to change the priority of the listener currently implemented via the config. Maybe the default value could be 0 or 1 instead.